test/e2e/QueueQuotaMgmt/QueueQuotaMgmt_suite_test.go

```diff
@@ -0,0 +1,104 @@
+/*
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package queuequotamgmt_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/onsi/ginkgo/reporters"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	"github.com/onsi/gomega"
+
+	"github.com/apache/incubator-yunikorn-k8shim/test/e2e/framework/configmanager"
+	"github.com/apache/incubator-yunikorn-k8shim/test/e2e/framework/helpers/k8s"
+)
+
+func init() {
+	configmanager.YuniKornTestConfig.ParseFlags()
+}
+
+var k = k8s.KubeCtl{}
+var oldConfigMap *v1.ConfigMap
+
+var _ = BeforeSuite(func() {
+	By("Enable basic scheduling config over config maps")
+	Ω(k.SetClient()).To(BeNil())
+	var c, err = k.GetConfigMaps(configmanager.YuniKornTestConfig.YkNamespace,
+		configmanager.DefaultYuniKornConfigMap)
+	Ω(err).NotTo(HaveOccurred())
+	Ω(c).NotTo(BeNil())
+
+	oldConfigMap = c.DeepCopy()
+	Ω(c).Should(BeEquivalentTo(oldConfigMap))
+	var configStr = "partitions:\n  -\n    name: default\n    placementrules:\n      - name: tag\n        " +
```

Comment:
Could you pls check if the config can be defined like: https://github.com/apache/incubator-yunikorn-core/blob/357d71fa290619e583b5da0484215c5d927807b7/pkg/scheduler/tests/scheduler_smoke_test.go#L41-L52

Reply:
Ok.